moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -198,12 +185,11 @@ void ServoCalcs::start()
 
   // TODO(adamp or andyz): after rebasing on upstream master, update this section. Maybe put the tf updates in a tiny
```

Comment:
Found this TODO comment while looking through the changes.

I added it, and am pretty sure it's ok to remove. At the time, I noticed the calculating the transforms below happen in 2 separate places but are identical lines and thought we might want to make it a function. I think its ok to leave as it or do the function, but we should probably remove the TODO comment

Reply:
done, thank you for the review :)